### DISCUSSION TITLE
Ошибка при сборки старой версий

### DISCUSSION THREAD

**Discussion by Murakumo-JP** [xtrime-ru/antizapret-vpn-docker#44] [Q&A]
Здраствуйте решил откатится на версию 3.1.0 так как на свежей версий у людей есть дисконеты от сервера, так как не знал как откатися назад через гит, переустановил систему на vps поставил докер и закинул проста папку на сервер со старой версий, при сборке получаю в конце ошибку, как решить проблему подскажите пожалуйста.  
![image](https://github.com/user-attachments/assets/257a14be-f0f9-4006-84ca-68ce9705beac)
Или подскажите как установить старую версию на прямую с гита я проста не селён в этом.

**Comment by Murakumo-JP** [comment]
Откатился через git checkout 51c7c9c вроде всё заработала

**Reply by xtrime-ru** [reply]
На будущее можно в docker-compose.yml вместо latest указать конкретную версию.  Либо скачать старую версию: `docker pull xtrime/antizapret-vpn:3.1.0`

**Reply by Murakumo-JP** [reply]
о как понял вас спасибо
